bl31/ehf.c

```diff
@@ -52,7 +52,6 @@ static int pri_to_idx(unsigned int priority)
 
 	idx = EHF_PRI_TO_IDX(priority, exception_data.pri_bits);
 	assert((idx >= 0) && (idx < exception_data.num_priorities));
-	assert(IS_IDX_VALID(idx));
```

Comment:
At this point, we have both the index an prioirty values here, so if at all, why not print the information here rather than duplicating error messages from the caller?

Reply:
At here we only can print out index value, but cannot print the interrupt number.